test/command_callback/test_remark_lint_command_callbacks.vader

```diff
@@ -0,0 +1,64 @@
+Before:
```

Comment:
Rebase your commits, and look at how linter tests are written now. If you move the functions back to where they were, you should be able to make this file a lot smaller. See `:help ale-development-linter-tests` in the latest version of ALE.